src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    line-height: 14px;
+    padding: $product-img-padding 0;
+
+    &-reviews {
+      margin: 0;
+      font-size: $font-S;
+      color: $page-text-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 4px;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 .stars__star:nth-last-child(1n + 2) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
According to the code style guide, it's better to keep all the constants and magic numbers in one place. Therefore, you might want to consider moving the pixel values (4px, 96px, 16px) to a separate variables file. This would make your code more maintainable and easier to read.